app/api_topology_test.go

```diff
@@ -43,14 +43,14 @@ func TestAPITopologyApplications(t *testing.T) {
 		// Let's not unit-test the specific content of the detail tables
 	}
 	{
-		body := getRawJSON(t, ts, fmt.Sprintf("/api/topology/applications/%s/%s", expected.ClientProcessID, expected.ServerProcessID))
+		body := getRawJSON(t, ts, fmt.Sprintf("/api/topology/applications/%s/%s", expected.ClientProcess1ID, expected.ServerProcessID))
 		var edge APIEdge
 		if err := json.Unmarshal(body, &edge); err != nil {
 			t.Fatalf("JSON parse error: %s", err)
 		}
 		want := render.AggregateMetadata{
-			"egress_bytes":  30,
-			"ingress_bytes": 300,
+			"egress_bytes":  10,
+			"ingress_bytes": 100,
```

Comment:
Yeah a bunch of tests in render/topologies_test.go check for that on the the process by name, container and container images topologies.